swaybar/main.c

```diff
@@ -466,6 +477,12 @@ int main(int argc, char **argv) {
 
 	int desired_output = atoi(argv[optind]);
 	struct output_state *output = registry->outputs->items[desired_output];
+	
+	window = window_setup(registry, output->width, 30, false);
```

Comment:
I wasn't really sure how to do this, as we need the window created before we can find the height of the text when no height is specified. The 30 being passed here is overwriten later before the window gets rendered anyway so it doesn't matter. It's a bit of a catch 22. I suppose I can change it to 0 to make it cleaner